packages/tasks/src/tasks/index.ts

```diff
@@ -131,6 +131,7 @@ export const TASKS_MODEL_LIBRARIES: Record<PipelineType, ModelLibraryKey[]> = {
 	"image-to-image": ["diffusers", "transformers", "transformers.js"],
 	"image-to-text": ["transformers", "transformers.js"],
 	"image-to-video": ["diffusers"],
+	"keypoint-detection": ["transformers"],
```

Comment:
What does this mean? Cause there's no keypoint detection pipeline in Transformers yet

Reply:
> we currently have SuperPoint and will soon have ViTPose

I think we should tag those models then before the PR is merged

Reply:
I have opened PRs to 30-40 models and some are merged already I think, see

also Sapiens model will have a lot of keypoint models soon (it was one unified model repo, I sent authors a script to automate separation)

<img width="508" alt="Screenshot 2024-08-29 at 14 02 42" src="https://github.com/user-attachments/assets/0d703c9f-0ee9-47c2-be5d-61a38aa37d59">

Reply:
Yes, I see 4 models now 👍